nav2_system_tests/src/costmap_filters/tester_node.py

```diff
@@ -192,12 +209,24 @@ def checkKeepout(self, x, y):
             return False
         return True
 
+    def checkSpeed(self, it, speed_limit):
```

Comment:
Can you add a comment block about what the speed test is doing?

Reply:
Done in https://github.com/ros-planning/navigation2/commit/34e694dbee7af9f805d2a7038b292466e44dc104#diff-d2306699cc23937cb8d74712a7b073fd7fd18c28ff00313035041cdd3d3989d8. Hope, this will be descriptive